public/docs/_examples/template-syntax/dart/lib/little_hero.dart

```diff
@@ -0,0 +1,14 @@
+library template_syntax.littleHero;
```

Comment:
We use _ separation, not camelCase, for [library names](https://www.dartlang.org/effective-dart/style/#do-name-libraries-and-source-files-using-lowercasewithunderscores).